coalib/parsing/DefaultArgParser.py

```diff
@@ -187,6 +187,10 @@ def default_arg_parser(formatter_class=None):
         help='filters `--show-bears` by the given languages')
 
     outputs_group.add_argument(
+        '--filter-by', action='append', nargs='+',
+        help='filter --show-bears by the filter given as argument.')
```

Comment:
@Makman2 That makes sense. Currently there's no direct way of knowing which filters are available unless user tries to filter bears with some wrong filter name.
One solution I can think of is to add one more option called `--show-filters` in `--show-bears` which will display all the available filters.
I'm confused b/w two of the following options:
`coala -B --show-filters`
or
`coala -B show-filters`

What's your opinion ?